docs/0.1/grid_on_splinter.md

```diff
@@ -0,0 +1,459 @@
+# Running Hyperledger Grid on Splinter
+
+This document shows how to set up a Grid-on-Splinter environment that runs in a
```

Comment:
I think we need to add a first sentence that defines Splinter. Perhaps something like this:

`Hyperledger Grid supports [Splinter](https://www.splinter.dev/) as a backend distributed ledger.`